src/ontology/mondo-edit.obo

```diff
@@ -430957,13 +431218,45 @@ is_a: MONDO:0035561 {source="https://orcid.org/0000-0001-5208-3432", source="htt
 relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/9449 ! PRNP
 relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/general/qc-single-child.sparql
 
+[Term]
+id: MONDO:0035639
+name: mixed phenotype acute leukemia with t(9;22)(q34.1;q11.2)
+subset: ordo_subtype_of_a_disorder
+synonym: "MPAL with t(9;22)(q34.1;q11.2); BCR-ABL1" EXACT [Orphanet:589534]
+xref: ICD10:C92.7 {source="MONDO:mondoIsNarrowerThanSource", source="Orphanet:589534"}
+xref: Orphanet:589534 {source="MONDO:equivalentTo"}
+is_a: MONDO:0020743 {source="https://orcid.org/0000-0001-5208-3432"} ! mixed phenotype acute leukemia
+relationship: RO:0004004 http://identifiers.org/hgnc/3765 ! FLT3
```

Comment:
I don't think RO:0004004 is in Mondo
[has material basis in somatic mutation in](https://www.ebi.ac.uk/ols/ontologies/ro/properties?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FRO_0004004)
![Screen Shot 2022-04-12 at 6 19 11 PM](https://user-images.githubusercontent.com/12737987/163080189-c24c31ae-49a1-4a12-acfc-25df1a2dc348.png)

I think the relation should be "disease has basis in dysfunction"